src/ray/core_worker/actor_manager.cc

```diff
@@ -162,8 +162,8 @@ void ActorManager::HandleActorStateNotification(const ActorID &actor_id,
                 << ", raylet_id: "
                 << ClientID::FromBinary(actor_data.address().raylet_id())
                 << ", num_restarts: " << actor_data.num_restarts();
-
-  if (actor_data.state() == gcs::ActorTableData::PENDING) {
+  if (actor_data.state() == gcs::ActorTableData::PENDING_DEPENDENCY_RESOLUTION ||
+      actor_data.state() == gcs::ActorTableData::PENDING_CREATION) {
```

Comment:
We should change the last `else` to `else if (ALIVE)`, and put this part in the last `else`.

Reply:
Actually, if no client cares these 2 states. Should we just stop publishing them?

Reply:
That's a good point, doesn't seem like they get used.